consensus/src/threshold_simplex/types.rs

```diff
@@ -364,8 +368,12 @@ impl<D: Digest> Read for Notarize<D> {
     }
 }
 
-impl<D: Digest> FixedSize for Notarize<D> {
-    const SIZE: usize = Proposal::<D>::SIZE + PartialSignature::SIZE + PartialSignature::SIZE;
+impl<D: Digest> EncodeSize for Notarize<D> {
+    fn encode_size(&self) -> usize {
+        self.proposal.encode_size()
+            + self.proposal_signature.encode_size()
```

Comment:
Can you make a todo to convert the `PartialSignature` encoding to use `varint` as well? I expect that to almost always be < 100?